content/api/commands/mount.md

```diff
@@ -46,7 +46,7 @@ the commands file. Below are examples to start with for your commands:
 ```js
```

Comment:
Can you update text on line 40 to mention `Cypress.Commands.add()`? Something like

> To use `cy.mount()`, add a [custom command](/api/cypress-api/custom-commands) to the commands file using [`Cypress.Commands.add()`](/api/cypress-api/custom-commands).